src/ontology/mondo.Makefile

```diff
@@ -289,6 +289,41 @@ sources/$(SOURCE_VERSION)/equivalencies.owl: | source_release_dir
 #sources/CTD_diseases.obo:
 #	curl -L -s http://ctdbase.org/reports/CTD_diseases.obo.gz  | gzip -dc | perl -npe 's@alt_id@xref@' > $@.tmp && mv $@.tmp $@
 
+# TODO: Gard mapped obsoletes
+# Working helper commands ------
```

Comment:
I added the makefile commands I needed to the "mondo analysis" section of the makefile.